jujugui/static/gui/src/app/components/profile/navigation/test-navigation.js

```diff
@@ -0,0 +1,109 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+var juju = {components: {}}; // eslint-disable-line no-unused-vars
+
+describe('Profile Navigation', function() {
+
+  beforeAll(function(done) {
+    // By loading this file it adds the component to the juju components.
+    YUI().use('profile-navigation', function() { done(); });
+  });
+
+  it('can render', () => {
+    const changeState = sinon.stub();
+    const output = jsTestUtils.shallowRender(
```

Comment:
I'd start putting rendering code in its own method here so that all tests can use that without too much repetition. We have too many cases of testing code never abstracted out, so maybe by doing that earlier even when it's not really required could help.